src/omdbapi.js

```diff
@@ -0,0 +1,20 @@
+const API_KEY = "6de06ef5"
+const API_BASE = "https://www.omdbapi.com/?apikey=" + API_KEY;
+
+export async function loadIMDBMovieRating (id, query = "&i=") {
```

Comment:
What is the purpose of declaring the `query` parameter with a default value, as a parameter. I can't see that it's used as an argument anywhere, so it's really just a variable, and IMO should be declared inside the function body.